### DISCUSSION TITLE
EA rpm packages not signed with gpg key...

### DISCUSSION THREAD

**Discussion by Dutchglory** [dbeaver/dbeaver#22361] [DBeaver Development]
EA rpm packages at [https://dbeaver.io/files/ea/](https://dbeaver.io/files/ea/) not signed with gpg key.  pleas add gpg key or sign packages with gpg key at [https://dbeaver.io/debs/](https://dbeaver.io/debs/)

thank you....

**Comment by LonwoLonwo** [comment]
Hello @Dutchglory 

Could you please check again?
